screen/pause_menu.h

```diff
@@ -14,5 +14,5 @@ class pause_menu : public screen {
 
     std::vector<button*> buttons;
 
-    virtual View_mode Run(sf::RenderWindow& window) override;
+    virtual View_mode Run(sf::RenderWindow& window, std::shared_ptr<Player> player) override;
 };
```

Comment:
`std::shared_ptr<Player> player` also add nullptr as a default to clarify code function